gui/storage/views.py

```diff
@@ -411,6 +412,7 @@ def dataset_create(request, fs):
             errno, errmsg = notifier().create_zfs_dataset(
                 path=str(dataset_name),
                 props=props)
+	    notifier().zfs_set_option(name = str(dataset_name),item = "volume_entity:comments",value = dataset_comments)
```

Comment:
Use spaces here too, also please make it PEP8 to conform with all the rest of the codebase, in this case means do not put spaces between equal signs in methods usage, and a space after comma.

e.g. name=str(dataset_name), item="volume...